assets/script2.js

```diff
@@ -0,0 +1,219 @@
+      if (questionIndex >= questions.length) {
+        quizCompleted();
+        createDiv.textContent = "You have finished the quiz! You got  " + score + "/" + questions.length + " correct!";
+        createDiv.setAttribute("style", "color:black; padding: 20px; border-top: 0px;");
+      } else {
+        displayQuestion(questionIndex);
+      }
+      questionContent.appendChild(createDiv);
+  }
+}
+
+function quizCompleted () {
+  heading.innerHTML = "";
+  questionContent.innerHTML = "";
+  seconds.textContent = timeLeft;
+
+  var newH1 = document.createElement("h1");
+  newH1.setAttribute("id", "finalH1");
+  newH1.textContent = "The Quiz is Over!";
+  questionContent.appendChild(newH1);
+
+  var pTag = document.createElement("p");
+  pTag.setAttribute("id", "pTag");
+  questionContent.appendChild(pTag);
+
+  if (timeLeft >=0) {
+    var timeScore = timeLeft;
+    var newPtag = document.createElement("p");
+    clearInterval(timeInterval);
+    pTag.textContent = "Your score is " + timeScore;
+    questionContent.appendChild(newPtag);
+  }
+
+  var nameInput = document.createElement("input");
+  nameInput.setAttribute("type", "text");
+  nameInput.setAttribute("id", "userInitials");
+  nameInput.textContent = "";
+  nameInput.placeholder = "Enter your name/initials here!";
+  nameInput.setAttribute("style", "border: 2px solid black; font-size: 25px; text-align: center; border-radius:20px; background-color: lightblue; color: red; font-weight: bold; margin: 20px; padding: 20px; width: 80%;");
+  questionContent.appendChild(nameInput);
+
+  var scoreSubmit = document.createElement("button");
+  scoreSubmit.setAttribute("type", "submit");
+  scoreSubmit.setAttribute("style", "font-size: 30px; border-radius: 20px; background-color: blue; color: white; padding: 10px; margin: auto; display: block;");
+  scoreSubmit.textContent = "Submit";
+  questionContent.appendChild(scoreSubmit);
+
+  scoreSubmit.addEventListener("click", function() {
+    var playerName = document.getElementById("userInitials").value;
+     {
+
+      var savedScore = {
+        initials: playerName,
+        score: timeScore
+      }
+      console.log(savedScore);
+      var savedScoreString = JSON.stringify(savedScore);
+      localStorage.setItem('savedScore', savedScoreString);
+
+      //var savedStringFromLocalStorage = localStorage.getItem('savedScore');
+      //var savedLocalStorageInfo = JSON.parse(savedStringFromLocalStorage);
+      //console.log(savedLocalStorageInfo);
+    }
+    highScores ();
+  });
+
+  function highScores () {
+    heading.innerHTML = "";
+    questionContent.innerHTML = "";
+    var newDiv = document.createElement("div");
+    newDiv.setAttribute("id", "highScoreDiv");
+    newDiv.nodeValue = nameInput, timeScore;
+    questionContent.appendChild(newDiv);
+
+    var storedInfoString = localStorage.getItem("savedScore");
+    var savedInfo = JSON.parse(storedInfoString);
+
+    document.getElementById("highScoreDiv").innerHTML = savedInfo;
```

Comment:
Here is why you are getting the [object object]. String in the HTML. You are setting the innterHTML to a javascript object. When you do this it calls the object's inherited `toString` method which for an object creates the string '[object object]'. You will need to assemble the string your self. So for this one you can use.

```javsacript
document.getElementById("highScoreDiv").innerHTML = "<p>" + savedInfo.initials + ":" + savedInfo.score + "</p>";

// or with the new string template syntax
document.getElementById("highScoreDiv").innerHTML = `<p>${savedInfo.initials}: ${savedInfo.score}</p>`
```
both should work.

Keep in mind the way you are storing and display this is assuming you are only showing the last score the user got. If you want to save many scores, you will have to change this to an array of objects or another type.